lib/src/block_parser.dart

```diff
@@ -713,8 +708,6 @@ class OrderedListSyntax extends ListSyntax {
 
 /// Parses paragraphs of regular text.
 class ParagraphSyntax extends BlockSyntax {
-  static final _reflinkDefinitionStart = new RegExp(r'[ ]{0,3}\[');
```

Comment:
Oops! I meant to use this in `lineStartsReflinkDefinition` on line 749 (742 on the right), for efficiency. Would you mind using this there instead of deleting it?